mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule SocketProxy.Mixfile do
     [
       app: :socket_proxy,
       version: "0.1.0",
-      elixir: "~> 1.5",
+      elixir: "~> 1.9",
```

Comment:
Should this be 1.10?

Reply:
Hm, I actually can't find concretely what this configuration option is supposed to mean, but I've been treating it something like "minimum compatible Elixir version". I think that's the case since I remember seeing warnings messages when compiling before along the lines of "dependency requires Elixir v. X but you're using Y".

So I set it to 1.9 because it uses `mix release` which was introduced in 1.9, and it doesn't use any 1.10 specific features, I don't think.